modules/packages/Sort.chpl

```diff
@@ -422,13 +422,16 @@ proc radixSortOk(Data: [?Dom] ?eltType, comparator) param {
 
 /*
 
-Sort the elements in a 1D rectangular array.  The choice of sorting
-algorithm used is made by the implementation.
+Sort the elements in a 1D rectangular array. After the call, the ``Data`` array
+will store elements in sorted order.
```

Comment:
How about this tweak?

Sort the elements in a 1D rectangular array ``Data``. After the call,
``Data`` will store elements in sorted order.